libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -223,4 +224,13 @@ private static Integer getAge(String dobValue) {
             return null;
         }
     }
+
+    public static Vot getVcVot(SignedJWT vc) throws UnrecognisedVotException {
```

Comment:
Should this replace usages of `UserIdentityService#getVot` - it seems basically identical except that it catches its own exceptions?

Reply:
We talked about this, and it's definitley a candidate for fixing up. The use cases are slightly different. This is used when we know and expect for a VC to potentially not contain a vot. The other is used when we're explicitly expecting a vot to exist. I suppose this version could be used and then null checks done on the result.